drivers/crc/crc_ccitt_sw.c

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright (c) Google LLC.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <crc16.h>
+
+u16_t crc_ccitt(u16_t seed, const u8_t *src, size_t len)
```

Comment:
Move this into drivers/crc/crc16_sw.c  : it is a software way of doing a 16bits crc.

Reply:
Hi @tbursztyka, thanks for the review.  Done.

Note that this may need refactoring later - I want to add support for the hardware CRC-16/CCITT unit in the SAMD21 but it'll still need software fallback for crc16_ansi etc.